src/content/advanced/cluster-management/node-pools-capi/index.md

```diff
@@ -0,0 +1,322 @@
+Also, the order in which we define the instance types in `instanceTypeOverrides` is important.
+The first instance type in the list that is available in the selected availability zone will be used.
+
+**Note:** Changing the `instanceTypeOverrides` value won't trigger a rolling update of the node pool.
+
+Using multiple instance types in a node pool has some benefits:
+
+- Together with spot instances, using multiple instance types allows better price optimization. Popular instance types tend to have more price adjustments. Allowing older-generation instance types that are less popular tends to result in lower costs and fewer interruptions.
+
+- Even without spot instances, AWS has a limited number of instances per type in each Availability Zone. It can happen that your selected instance type is temporarily out of stock just in the moment you are in demand of more worker nodes. Allowing the node pool to use multiple instance types reduces this risk and increases the likelihood that your node pool can grow when in need.
+
+## Node pools and autoscaling {#autoscaling}
+
+With node pools, you set the autoscaling range per node pool (suppported on AWS EC2 (CAPA) clusters only). The Kubernetes cluster autoscaler has to decide which node pool to scale under which circumstances.
+
+If you assign workloads to node pools as described [above](#assigning-workloads) and the autoscaler finds pods in `Pending` state, it will decide based on the node selectors which node pools to scale up.
+
+In case there are workloads not assigned to any node pools, the autoscaler may pick any node pool for scaling. For details on the decision logic, please check the upstream FAQ for [AWS](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md).
+
+## On-demand and spot instances {#on-demand-spot}
+
+{{< tabs >}}
+{{< tab id="cluster-capa-ec2" for-impl="capa_ec2" >}}
+
+Node pools can make use of [Amazon EC2 Spot Instances](https://aws.amazon.com/ec2/spot/). You just need to enable it and select the max price to pay on the node pool definition.
+
+```yaml
+metadata:
+  description: "my cluster"
+  name: test-cluster
+  organization: giantswarm
+nodePools:
+  pool0:
+    maxSize: 2
+    minSize: 2
+    spotInstances:
+      enabled: true
+      maxPrice: 1.2
+```
+
+{{< /tab >}}
+{{< tab id="cluster-capz-azure-vms" for-impl="capz_vms" >}}
+
+This is currently not supported.
+
+{{< /tab >}}
+{{< /tabs >}}
+
+## Limitations
+
+- Clusters without worker nodes (= without node pools) cannot be considered fully functional. In order to have all
+required components scheduled, worker nodes are required. For that reason, we deactivate any monitoring and alerts for
+these clusters and don't provide any proactive support.
+
+### Things to keep in mind
+
+{{< tabs >}}
+{{< tab id="cluster-capa-ec2" for-impl="capa_ec2" >}}
```

Comment:
```suggestion
{{< tabs >}}
{{< tab id="general" title="General" >}}

- Clusters without worker nodes (= without node pools) cannot be considered fully functional. In order to have all
required components scheduled, worker nodes are required. For that reason, we deactivate any monitoring and alerts for
clusters without worker nodes and don't provide any proactive support.

{{< /tab >}}
{{< tab id="cluster-capa-ec2" for-impl="capa_ec2" >}}
```

Reply:
and please ensure that all `tab id="..."` are unique (= don't repeat `id="cluster-capa-ec2"` but use `limitations-capa-ec2`) so that the tab-switching logic works i.e. switches all equally-titled tabs at once on the page.